### PR TITLE
update scalingo-22

### PR DESCRIPTION
**Pull request by curzolapierre**

### CODE REVIEW

File: bin/compile [hunk 1]
```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 
 set -e
+
+if [ -n "$DEBUG" ]; then
```
Comment (Soulou) [inline bin/compile:5]:
```suggestion
if [ -n "$BUILDPACK_DEBUG" ]; then
```

Comment (Soulou) [inline bin/compile:5]: Already present here: https://github.com/Scalingo/php-buildpack/blob/d560c1a1934daca2cfbbc8ef5af810e79d7488f5/bin/compile#L24-L26

File: bin/compile [hunk 2]
```diff
@@ -295,8 +300,12 @@ status "Bundling PHP ${PHP_VERSION}"
 
 # Install libraries also on build container, so PHP doesn't complain about missing
 # dynamic libraries when running it during the slug compilation.
-fetch_package "$PHP_BASE_URL" "libmcrypt-${mcrypt_version}" /app/vendor/libmcrypt > /dev/null
-fetch_package "$PHP_BASE_URL" "libzip-${zip_version}" /app/vendor/libzip > /dev/null
+if [ "$(php_api_version)" = "20170718" -o "$(php_api_version)" = "20180731" -o "$(php_api_version)" = "20190902" ] ; then
```
Comment (Soulou) [inline bin/compile:303]: Condition is not understandable, make a helper function.

File: bin/compile [hunk 3]
```diff
@@ -295,8 +300,12 @@ status "Bundling PHP ${PHP_VERSION}"
...
+  fetch_package "$PHP_BASE_URL" "libmcrypt-${mcrypt_version}" /app/vendor/libmcrypt > /dev/null
+fi
+if [ "$(php_api_version)" = "20170718" ] ; then
```
Comment (Soulou) [inline bin/compile:306]: Semantic is not clear, make a helper function

**Review by Soulou — CHANGES_REQUESTED** [review]

**Comment by SCedricThomas** [comment]
Why do we include PHP 7.3 for mcrypt bundling as it is no longer available as a standard php extension: 
https://www.php.net/manual/en/mcrypt.installation.php 
https://github.com/Scalingo/php-buildpack/blob/dd52e9fae2da9708b94844400661d64a17eabaea/bin/compile#L343-L346

**Comment by Soulou** [comment]
If deployments work for scalingo-22 and the different php version, LGTM

**Pull request merged**